src/search/containers/CurrentSearchPage.js

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux';
+import SearchPage from '../components/SearchPage';
+import { setQuery } from '../actions';
+
+function getVisibleAddons(addons, query) {
+  const matches = [];
+  for (const slug in addons) {
+    if (addons[slug].title.indexOf(query) >= 0) {
```

Comment:
>  maybe Chakra decides to return -0.3333333333333....
I think that would break most of the JS on the internet if that ever happened. 

Checking against `-1` is more direct because it has an explicit meaning `>= 0` is more roundabout.